packages/react-api/src/ledger.ts

```diff
@@ -5,17 +5,32 @@
 import { Ledger } from '@polkadot/ui-keyring';
 import uiSettings from '@polkadot/ui-settings';
 
-import chains from '@polkadot/ui-settings/defaults/chains';
-
+import { assert } from '@polkadot/util';
 import { api } from './Api';
+import networks from '@polkadot/networks';
+
+function getGenesis (name: string): string {
+  const network = networks.find(({ network }) => network === name);
+
+  assert(network && network.genesisHash[0], `Unable to find genesisHash for ${name}`);
+
+  return network.genesisHash[0];
+}
 
-const ALLOWED_CHAINS = chains.kusama;
+const KUSAMA_GENESIS = getGenesis('kusama');
+
+const POLKADOT_GENESIS = getGenesis('polkadot');
+
+const ALLOWED_CHAINS: [string, 'kusama' | 'polkadot'][] = [
+  [KUSAMA_GENESIS, 'kusama'],
+  [POLKADOT_GENESIS, 'polkadot']
```

Comment:
What about Azalea and Nikau?

Reply:
ya.. I am looking at https://github.com/polkadot-js/common/blob/master/packages/networks/src/index.ts package to see how they use the KUSAMA and POLKADOT networks genesis hash.. I am also trying to understand when this Ledger is been used..

Reply:
Right now as we don't support hardware type...we won't add support for Azalea n Nikau at this stage.